stories/core/Table.stories.tsx

```diff
@@ -1088,6 +1089,12 @@ ControlledState.args = { isSelectable: true };
 export const Full: Story<TableProps> = (args) => {
   const { columns, data, ...rest } = args;
 
+  const [hoveredRow, setHoveredRow] = useState<
+    { name: string; description: string } | undefined
```

Comment:
If we expose `Row`, we'll be able to save the ids instead of the whole data

Reply:
Saving index because id comes from inside of react-table and can't be really used outside of it (except if custom id function is passed).